doc/release/multilocale.rst

```diff
@@ -5,6 +5,8 @@ Multilocale Chapel Execution
 ============================
 
 This document outlines the steps to get started with multi-locale Chapel.
+Steps 2-3 describe how to build a multilocale Chapel and steps 4-6 cover
+compiling and running multilacale Chapel programs.
```

Comment:
Putting on my Paul hat, 'multilocale' is mis-spelled.  There should be a comma before "and steps 4-6" since it has its own subject.